NineChroniclesUtilBackend.Store/Util/IStateJsonConverter.cs

```diff
@@ -0,0 +1,46 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Nekoyume.Model.State;
+using Libplanet.Common;
+using Bencodex;
+using Bencodex.Types;
+
+namespace NineChroniclesUtilBackend.Store.Util;
+
+public class IStateJsonConverter : JsonConverter
```

Comment:
Since `I`-prefix identifier is reserved for interfaces, it should be `StateJsonConverter` instead. If you apply this suggestion, you should update its filename too.

```suggestion
public class StateJsonConverter : JsonConverter
```

Reply:
!!